PROTOCOL.md

```diff
@@ -3307,3 +3317,16 @@ Example:
   "expirationTime": "2021-11-12T00:12:29.0Z"
 }
 ```
+
+# Names
+
+Share, schema, and table objects are identifiable by names. To ensure compatibility and avoid issues across different sharing servers, the following limitations apply for object names:
```

Comment:
nit: should it be "Share, Schema, and Table object ...", capital on the initial letter?